native/packages/react-native-bpk-component-card/src/__snapshots__/BpkCard-test.android.js.snap

```diff
@@ -27,13 +27,13 @@ exports[`Android BpkCard should render correctly 1`] = `
           Object {
             "backgroundColor": "rgb(255, 255, 255)",
             "elevation": 2,
-            "shadowColor": "rgb(37, 32, 51)",
+            "shadowColor": undefined,
```

Comment:
This doesn't look good - perhaps we should conditionally apply these styling props based on `Platform.os`? 

Saying that I reckon we probably do this another places ... thoughts?

Reply:
I think we could go with one of three ways:
1. create a getter function that based on the OS returns the correct properties
2. have a default for everything if a value is not passed through e.g. `{shadowColor: token || defaultValue}`
3. leave it as it is, Android will ignore that property anyway as it's iOS only (https://facebook.github.io/react-native/docs/shadow-props.html#shadowcolor)

wdyt?

Reply:
Yeah perhaps option 1 & 2 are doing more work than is actually required - especially given that they are ignored on android. Ok lets leave as as (option 3) :)